src/app/pages/LiveRadioPage/index.jsx

```diff
@@ -95,10 +95,10 @@ const LiveRadioPage = ({ pageData }) => {
     translations,
   );
 
-  const { enabled } = useToggle('liveRadioSchedule');
-  // onLiveRadioPage is enabled on Persian to render the schedule for bbc_dari_radio
-  // however bbc_persian_radio should not show the schedule
-  const showSchedule = enabled && radioScheduleOnPage;
+  const { enabled, value } = useToggle('liveRadioSchedule');
+
+  const showSchedule =
```

Comment:
is this necessary?

Reply:
Which section?

Reply:
Ah I didn't realise local toggles are not service specific